spec/features/supplier_login_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature "Supplier Login" do
+  let(:default_supplier_attributes) { {name: "Supplier X", slug: "supplier-x", description: "A store for all of you disaster relief needs", retired: false, address: "101 William White Blvd, Pueblo, Co 80111" }}
```

Comment:
Line is too long. [210/80]<br>Space missing inside }.<br>Space inside { missing.